front/components/sparkle/AppLayout.tsx

```diff
@@ -356,24 +425,29 @@ export default function AppLayout({
         )}
 
         {!hideSidebar && (
-          <div className="hidden lg:fixed lg:inset-y-0 lg:z-0 lg:flex lg:w-80 lg:flex-col">
-            {loaded && (
-              <NavigationBar
-                owner={owner}
-                subscription={subscription}
-                subNavigation={subNavigation}
-                topNavigationCurrent={topNavigationCurrent}
-              >
-                {loaded && navChildren && navChildren}
-              </NavigationBar>
-            )}
-          </div>
+          <Transition.Root show={laptopNavOpen} as={Fragment}>
```

Comment:
ooc what do we need a transition here? It's not really animated is it?

Reply:
Nop it's not. Mainly used it for ease of implementation.

Reply:
But will use conditional rendering 👍🏼